server/src/test/java/org/opensearch/common/unit/FuzzinessTests.java

```diff
@@ -138,6 +140,26 @@ public void testParseFromXContent() throws IOException {
 
     }
 
+    public void testFuzzinessValidationWithStrings() throws IOException {
+        String[] invalidStrings = new String[] { "+++", "asdfghjkl", "2k23" };
+        XContentBuilder json = jsonBuilder().startObject().field(Fuzziness.X_FIELD_NAME, randomFrom(invalidStrings)).endObject();
+        try (XContentParser parser = createParser(json)) {
+            assertThat(parser.nextToken(), equalTo(XContentParser.Token.START_OBJECT));
+            assertThat(parser.nextToken(), equalTo(XContentParser.Token.FIELD_NAME));
+            assertThat(parser.nextToken(), equalTo(XContentParser.Token.VALUE_STRING));
+            IllegalArgumentException e = expectThrows(IllegalArgumentException.class, () -> Fuzziness.parse(parser));
+            assertThat(e.getMessage(), containsString("Invalid fuzziness value:"));
+        }
+        json = jsonBuilder().startObject().field(Fuzziness.X_FIELD_NAME, "AUTO:").endObject();
+        try (XContentParser parser = createParser(json)) {
+            assertThat(parser.nextToken(), equalTo(XContentParser.Token.START_OBJECT));
+            assertThat(parser.nextToken(), equalTo(XContentParser.Token.FIELD_NAME));
+            assertThat(parser.nextToken(), equalTo(XContentParser.Token.VALUE_STRING));
+            OpenSearchParseException e = expectThrows(OpenSearchParseException.class, () -> Fuzziness.parse(parser));
+            assertThat(e.getMessage(), containsString("failed to find low and high distance values"));
```

Comment:
what's the benefit of `containsString()`  comparing to `string. contains()`?

Reply:
was following similar `asserts` in [FuzzyQueryBuilderTests.java](https://github.com/opensearch-project/OpenSearch/blob/ac3351cc54e6fa1934fe0c15f3d1e3cdbef153b7/server/src/test/java/org/opensearch/index/query/FuzzyQueryBuilderTests.java#L120). would it be better to just use `String.contains()`?

Reply:
Maybe use startsWith() instead of containsString() if it's validating on the prefix.

Reply:
resolved in [8542176](https://github.com/opensearch-project/OpenSearch/pull/5805/commits/8542176477525291c2f4c1b6ab5c042537a7eaad)